app/index.js

```diff
@@ -53,7 +53,7 @@ app.on("ready", function () {
     creatorFunc()
         .then(creator.check)
         .then(function () {
-            localize.load(app.getLocale());
-            abrApp = new AbrApplication(osxOpenFilePaths);
+            var localizer = new Localizer(app.getLocale());
```

Comment:
I'm wondering why don't you require and instanciate Localizer directly in abr-application ?